ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productteaser/v1/productteaser/actions.html

```diff
@@ -17,11 +17,11 @@
     <div class="productteaser__cta" data-sly-test="${product.callToAction != null}">
         <button data-sly-test="${product.callToAction == 'add-to-cart'}" data-action="addToCart" data-item-sku="${product.sku}"
                 class="button__root_highPriority button__root clickable__root button__filled" type="button">
-            <span class="button__content"><span>Add to Cart</span></span>
+            <span class="button__content"><span>${properties.ctaText @ i18n}</span></span>
```

Comment:
I18n for something entered by the user? Will that even work?

Also there is no default fallback. If the user adds the component to the page but not enters a call to action text this will render an empty button. I suggest to use something like `${properties.text || 'Add to Cart' @ i18n}`.

Reply:
Sure `i18n` will work, as long as the text entered by the author is in the translation dictionaries. I'm fine to add the default.